rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -33,7 +33,7 @@ The following list describes the key features of ROSA:
 
 * **Upgrade handling**: The commands `rosa upgrade cluster` and `rosa upgrade account` roles are seamless and backward compatible with all the older ROSA versions and older {product-title} clusters. When you run these commands, the old {product-title} clusters and their relevant STS roles and policies are updated to the latest versions.
 
-* **New AWS roles supporting the ROSA UI**: The tools to manage the `ocm-role` and `user-role` and use the ROSA provisioning UI are now available in the ROSA CLI as of version 1.2.0. The `ocm-role` and `user-role` are also instrumental in facilitating ROSA STS upgrades done from the {cluster-manager-first} UI only.
+* **New AWS roles supporting the ROSA UI**: The tools to manage the OpenShift Cluster Manager role and user role and use the ROSA provisioning UI are now available in the ROSA CLI as of version 1.2.0. The OpenShift Cluster Manager role and user roles are also instrumental in facilitating ROSA STS upgrades done from the {cluster-manager-first} UI only.
```

Comment:
You can revert this change, I can make it in the what's new PR as to not create a conflict.